appsec/src/extension/user_tracking.c

```diff
@@ -61,6 +67,88 @@ static PHP_FUNCTION(set_user_wrapper)
     _ddtrace_set_user(INTERNAL_FUNCTION_PARAM_PASSTHRU);
 }
 
+static void _fire_user_event(void)
+{
+    dd_tags_set_user_event_triggered();
+    dd_trace_emit_asm_event();
+    dd_tags_set_sampling_priority();
```

Comment:
Just a reminder that on your other PR, these two are merged together (It confused me...)

Reply:
I will adapt the second one merged